app/rtc_compare.py

```diff
@@ -727,6 +747,11 @@ def _print_first_value_diff(image_1, image_2, prefix):
        prefix: str
             Prefix to the message printed to the user
     """
+    failed_pixel_index = ~np.isclose(
+        image_1, image_2, atol=RTC_S1_PRODUCTS_ERROR_ABS_TOLERANCE,
+        rtol=RTC_S1_PRODUCTS_ERROR_REL_TOLERANCE, equal_nan=True)
+    failed_pixel_ratio = failed_pixel_index.sum() / failed_pixel_index.size
```

Comment:
Thank you, @seongsujeong , I guess we'll have to use this `failed_pixel_ratio` indeed. Could you please use the number of valid pixels in the denominator (you can use np.isfinite() for example) rather than `failed_pixel_index.size`. You may need to update `RTC_S1_PRODUCTS_FAILED_PIXEL_RATIO_TOLERANCE`